src/apis/applicant_api.py

```diff
@@ -0,0 +1,41 @@
+from flask import Blueprint, request
+from flask_restful import Api, Resource
+import os
+
+# Creates a new blueprint
+file_upload = Blueprint('file_upload', __name__)
+api = Api(file_upload)
+
+
+# Class containing all endpoints for applicants
+class ApplicantResource(Resource):
+
+    def get(self):
+        return 'Hello Upload Page!'
+
+    def post(self):
+        if 'pdfFile' not in request.files:
+            return 'No file part found in POST request.', 400
+
+        pdf_file = request.files['pdfFile']
+
+        # Handle the uploaded file (save, process, etc.)
+        print('File name: ' + pdf_file.filename)
+
+        # Change filename to the applicant id
+        # TODO: Generate UUID for applicant id somewhere
```

Comment:
The uuid of the applicant is set in the BusinessObject abstract class when initialising a Applicant instance. If this resolves the TODO and the concern.

Reply:
Yes, this resolves my TODOs. Since these classes are not on the main branch yet, I forgot about them.